app/controllers/videos_controller.rb

```diff
@@ -1,58 +1,9 @@
 class VideosController < ApplicationController
-  before_action :set_video, only: [:show, :edit, :update, :destroy]
-
-  # GET /videos
-  def index
-    @videos = Video.all
-  end
-
-  # GET /videos/1
   def show
-  end
+    @video = Video.find_by(slug: params[:slug])
 
-  # GET /videos/new
-  def new
-    @video = Video.new
+    @html_id  = "page"
+    @body_id  = "video"
+    @title    = @video.title
   end
-
-  # GET /videos/1/edit
-  def edit
-  end
-
-  # POST /videos
-  def create
-    @video = Video.new(video_params)
-
-    if @video.save
-      redirect_to @video, notice: 'Video was successfully created.'
-    else
-      render :new
-    end
-  end
-
-  # PATCH/PUT /videos/1
-  def update
-    if @video.update(video_params)
-      redirect_to @video, notice: 'Video was successfully updated.'
-    else
-      render :edit
-    end
-  end
-
-  # DELETE /videos/1
-  def destroy
-    @video.destroy
-    redirect_to videos_url, notice: 'Video was successfully destroyed.'
-  end
-
-  private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_video
-      @video = Video.find(params[:id])
-    end
-
-    # Only allow a trusted parameter "white list" through.
-    def video_params
-      params.require(:video).permit(:title, :subtitle, :content, :tweet, :summary, :tweet, :summary, :image, :image_description, :content_format, :quality, :slug, :year, :month, :day, :duration, :published_at, :vimeo_id)
-    end
```

Comment:
This was a good catch, deleting all the CRUD from the public site. It was sloppy of me to leave it behind.

Though, we should have it over in the admin section. I'll make a separate issue for that rather than growing the scope of this one.